### ISSUE TITLE
Recent rpi3 image does not work correctly

### ISSUE THREAD

**Issue by akhenakh** [therecipe/qt#666]
After pulling recent rpi3 image, my program does not compile anymore
`qtdeploy -docker build rpi3`

```
moc.cpp:10:19: fatal error: QCamera: No such file or directory
 #include <QCamera>
                   ^
compilation terminated.
```
moc.cpp
```


#define protected public
#define private public

#include "moc.h"
#include "_cgo_export.h"

#include <QByteArray>
#include <QCamera>
#include <QCameraImageCapture>
#include <QChildEvent>
#include <QDBusPendingCall>
#include <QDBusPendingCallWatcher>
#include <QEvent>
#include <QGraphicsObject>
#include <QGraphicsWidget>
#include <QLayout>
#include <QList>
#include <QMediaPlaylist>
#include <QMediaRecorder>
#include <QMetaMethod>
#include <QObject>
#include <QOffscreenSurface>
#include <QPaintDevice>
#include <QPaintDeviceWindow>
#include <QPdfWriter>
#include <QQuickItem>
#include <QRadioData>
#include <QSignalSpy>
#include <QString>
#include <QTime>
#include <QTimer>
#include <QTimerEvent>
#include <QWidget>
#include <QWindow>


class QmlBridge8f16db: public QObject

```

Images:
```
therecipe/qt                        rpi3                be38ff7ce27a        2 days ago          11.2GB
```
Same program compile fine with:
```
therecipe/qt                        linux               317531b98398        3 days ago          8.02GB
```

**Comment by therecipe** [comment]
Hey

I hope https://github.com/therecipe/qt/commit/c963a694b534bfcbfa5bc783f0528e0e46321c26 fixes this, but I haven't had time to reproduce the issue to really verify it.

**Comment by therecipe** [comment]
Mh, the changes prevented qtdeploy from creating the necessary cgo files for qtrcc and qtmoc.
But this should be fixed now with: https://github.com/therecipe/qt/commit/97cb6e94da47b8e55ec5c7d4fcb94ff1617a4ddd https://github.com/therecipe/qt/commit/618b3fc25aba228db08c70f1481d7e97a05aa028

**Comment by akhenakh** [comment]
solved in therecipe/qt:rpi3 29208812cd5f thank you
